README.md

```diff
@@ -5,9 +5,21 @@ A discord bot created for use in a personal discord server.
 - TypeScript
 - Node.JS v14
 
-# Setup
-This setup is only for macOS/Ubuntu users at the moment.
+# macOS/Ubuntu Setup
```

Comment:
Ideally we should have a "Setup" section that includes both environment setups and not creating individual setups.

I would change this back to "Setup" and create subsections for windows/mac/ubuntu.

Since installing TS, nodemon, and general install should work for all environments, just add a windows subsection and then the final install commands originally written for setup.

Reply:
Ignore this since you should just move the windows instructions into the setup section I told you about in the following review